pkg/rulesConfig/defaultRules/10-ensure-sca-scanner.go

```diff
@@ -111,12 +122,12 @@ func githubErrorsRule10(githubData map[string]*githubConnector.GithubOwner) ([]*
 	return schemaErrors, nil
 }
 
-func gitlabErrorsRule10(gitlabData map[string]*gitlabConnector.GitlabGroup) ([]*SchemaError, error) {
+func gitlabErrorsRule10(gitlabData map[string]*gitlabConnector.GitlabGroup, sharedRegexExpressions []string) ([]*SchemaError, error) {
 	schemaErrors := make([]*SchemaError, 0)
 
 	for _, group := range gitlabData {
 		for _, project := range group.Projects {
-			foundScaScanner, err := findScaScannerRule10(project)
+			foundScaScanner, err := findGitlabScaScannerRule10(project, sharedRegexExpressions)
```

Comment:
Can't comment on line 139 - but `CiCdPlatform` could be either gitlab-ci or jfrog pipelines.